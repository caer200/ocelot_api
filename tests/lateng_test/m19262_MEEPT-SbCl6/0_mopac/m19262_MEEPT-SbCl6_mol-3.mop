CHARGES

 
Sb	4.456715 +1 13.723951 +1 -1.156882 +1
Cl	4.573991 +1 11.383315 +1 -0.911852 +1
Cl	4.319920 +1 16.089390 +1 -1.421961 +1
Cl	2.084806 +1 13.563526 +1 -1.147610 +1
Cl	6.807841 +1 13.858787 +1 -1.146418 +1
Cl	4.416017 +1 13.998741 +1 1.188095 +1
Cl	4.464697 +1 13.524551 +1 -3.507778 +1
