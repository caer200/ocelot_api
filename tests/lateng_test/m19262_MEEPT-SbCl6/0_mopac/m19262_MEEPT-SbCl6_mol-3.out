 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 21:32:10 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Sb          4.45671500  *  13.72395100  *  -1.15688200  *
     2      Cl          4.57399100  *  11.38331500  *  -0.91185200  *
     3      Cl          4.31992000  *  16.08939000  *  -1.42196100  *
     4      Cl          2.08480600  *  13.56352600  *  -1.14761000  *
     5      Cl          6.80784100  *  13.85878700  *  -1.14641800  *
     6      Cl          4.41601700  *  13.99874100  *   1.18809500  *
     7      Cl          4.46469700  *  13.52455100  *  -3.50777800  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Sb          4.4567     13.7240     -1.1569
     2        Cl          4.5740     11.3833     -0.9119
     3        Cl          4.3199     16.0894     -1.4220
     4        Cl          2.0848     13.5635     -1.1476
     5        Cl          6.8078     13.8588     -1.1464
     6        Cl          4.4160     13.9987      1.1881
     7        Cl          4.4647     13.5246     -3.5078

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: Sb Cl6  =     7 atoms



      MOLECULAR POINT GROUP   :   Ci  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1         Sb      2      3      4      5      6      7
      2         Cl      1
      3         Cl      1
      4         Cl      1
      5         Cl      1
      6         Cl      1
      7         Cl      1

     Ion Atom No.  Type    Charge
 
    1       1     Sb       -1

          COMPUTED CHARGE ON SYSTEM: -1

          COMPUTED CHARGE ON SYSTEM:  -1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
